### DISCUSSION TITLE
Hotkeys behave differently

### DISCUSSION THREAD

**Discussion by ShahriarKh** [spacedriveapp/spacedrive#1817] [General]
I noticed that some hotkeys and keyboard shortcuts are behaving differently than the OS native functionality.
Since space drive is cross-platform, I opened a discussion instead of an issue to see if it was intentional or not.
Examples:

+ **Hold <kbd>shift</kbd>, click file A, then click file Z**
Expected: All files from A to Z should be selected.
Actual: Only A and Z are selected.

+ **Hold <kbd>ctrl</kbd>, click file A, then click file B**
Expected: A and B should be selected.
Actual: Only B is selected (the hotkey doesn't work)

+ **Hold <kbd>ctrl</kbd>, scroll the mousewheel**
Expected: Change zoom level
Actual: Nothing happens

**Comment by utkubakir** [comment]
Are these Windows/Linux shortcuts? None of these are a thing on macOS as far as I know.

**Reply by ShahriarKh** [reply]
Yeah Windows shortcuts.
https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec
